tst/EnergyPlus/unit/HeatBalanceSurfaceManager.unit.cc

```diff
@@ -3135,7 +3135,7 @@ TEST_F(EnergyPlusFixture, HeatBalanceSurfaceManager_TestInterzoneRadFactorCalc)
     state->dataGlobal->NumOfZones = 2;
     state->dataMaterial->TotMaterials = 1;
     state->dataHeatBal->TotConstructs = 1;
-    state->dataViewFactor->NumOfSolarEnclosures = 2;
+    state->dataViewFactor->NumOfSolarEnclosures = 3;
```

Comment:
Extended this unit test to cover the real defect here (interior window with numEnclosres>numZones).

Reply:
:+1: